### ISSUE TITLE
order method missing after discover in custom route

### ISSUE THREAD

**Issue by ludoo0d0a** [WP-API/node-wpapi#231]
After the discovery, I can reach my custom routes using API.
As mentioned in "Query Parameters & Filtering Custom Routes" section of the doc, .perPage and .page works is present, but not .order, .orderBy.
Since I do not use registerRoute but discover, how can I add them as supported methods ?

In the discovery json, order and orderby methods are anyway present in endpoints GET.
```js
                     "order": {
                        "required": false,
                        "default": "desc",
                        "enum": ["asc", "desc"],
                        "description": "Order sort attribute ascending or descending."
                    },
                    "orderby": {
                        "required": false,
                        "default": "date",
                        "enum": ["date", "id", "include", "title", "slug", "menu_order"],
                        "description": "Sort collection by object attribute."
                    },
```

Thanks

**Comment by ludoo0d0a** [comment]
In fact, order, orderby (and offset) are not available in a simple case with posts : 

```js
    var wp = new WPAPI({endpoint:config.wp.endpoint});
    var posts = wp.posts();
    //ok:
    posts.perPage; 
    posts.page; 
    posts.filter;
    posts.category
    posts.tag;
    posts.param;
    //ko:
    posts.order;
    posts.orderby;
    posts.offset;
```

**Comment by kadamwhite** [comment]
You are correct that these were not available: issue #178 tracked adding `order`, `orderby` and other query methods that occur on many endpoints to WPRequest. While that issue has been closed, the library has not been released since the change landed. We just now completed a release to NPM, however, so `npm install wpapi@latest --save` and you should see all these methods available with discovery-generated handlers.

Do note that it's going to be `orderby`, not `orderBy`, as `orderby` lower-case is the name of the parameter and we only do our (admittedly dubious) method renaming for `snake_cased` parameters.
